_pages/algorithm-development/source-control-management.md

```diff
@@ -216,51 +222,51 @@ __I revoked access to the Algorithmia's Github OAuth application, and now none o
 
 
-### Hosting Source Code on Bitbucket
+### Hosting Source Code on Bitbucket Cloud
 
-By hosting your algorithm's source code on Bitbucket, you can take advantage of Bitbucket's rich set of developer features, such as pull requests, and also ensure that access to your source code is carefully mediated.
+By hosting your algorithm's source code on Bitbucket Cloud, you can take advantage of Bitbucket Cloud's rich set of developer features, such as pull requests, and also ensure that access to your source code is carefully mediated.
 
 If you're new to Git, we recommend [this tutorial series](https://try.github.io/) by GitHub.
 
-**Web IDE Support** At this time we do not support editing source code in our web app for Bitbucket-hosted algorithms. 
+**Web IDE Support** At this time we do not support editing source code in our web app for Bitbucket Cloud-hosted algorithms. 
 {: .notice-info}
 
-#### Connecting Your Algorithmia & Bitbucket Accounts
+#### Connecting Your Algorithmia & Bitbucket Cloud Accounts
 
-To connect your Algorithmia and Bitbucket accounts, simply select the Bitbucket instance you wish to use when creating your algorithm:
+To connect your Algorithmia and Bitbucket Cloud accounts, simply select the Bitbucket Cloud instance you wish to use when creating your algorithm:
 
-<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/scm_create_bitbucket_algorithm_unauthorized.png" alt="Authorization prompt for Bitbucket repository host" class="screenshot img-sm">
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/scm_create_bitbucket_algorithm_unauthorized.png" alt="Authorization prompt for Bitbucket Cloud repository host" class="screenshot img-sm">
 
-If you haven't yet connected your Bitbucket account, you will be prompted to do so. By connecting your Bitbucket account, you will give us access to create repositories on your behalf, which will house your algorithm's source code.
+If you haven't yet connected your Bitbucket Cloud account, you will be prompted to do so. By connecting your Bitbucket Cloud account, you will give us access to create repositories on your behalf, which will house your algorithm's source code.
 
-#### Creating Algorithms with Bitbucket
+#### Creating Algorithms with Bitbucket Cloud
 
-Once you've connected your Bitbucket account to Algorithmia, you will be able to host algorithm source code in Bitbucket.
+Once you've connected your Bitbucket Cloud account to Algorithmia, you will be able to host algorithm source code in Bitbucket Cloud.
 
-You can customize two aspects of the Bitbucket repositories that are created for your algorithms: the repository's owner, and its name.
+You can customize two aspects of the Bitbucket Cloud repositories that are created for your algorithms: the repository's owner, and its name.
```

Comment:
plurality conflict, change to:
You can customize two aspects of any Bitbucket Cloud repository that's created for an algorithm: ...